test/web-platform-tests/to-upstream/dom/collections/HTMLCollection-iterator.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<meta charset="utf-8">
```

Comment:
Not required, but it's always nice to add a few link rel=help to the spec links we discussed earlier, and you can add a link rel=author to get some form of lasting credit :)